### PR TITLE
[NEW] User gets UI feedback when message is pinned or unpinned

### LINKED ISSUE #13869: No user feedback when meassage is pinned or unpinned
### Description:

There is no user feedback telling the user that his message has been pinned or unpinned

### Steps to reproduce:

1. Select a message to be pinned or unpinned if already pinned
2. Click on three dots in the message (Open the message actions menu)
3. Click on pin message or unpin message.

### Expected behavior:

The user should get UI feedback that his message has been pinned or unpinned otherwise failure is shown.

### Actual behavior:

No feedback is given to the user

### PR DESCRIPTION
**Pull request by ashwaniYDV**
Closes #13869

**When message is pinned.**
Screenshot from 2019-03-23 14-19-54

**When message is unpinned.**
Screenshot from 2019-03-23 14-20-05

### CODE REVIEW

**Comment by ashwaniYDV** [comment]
@ggazzo Sir, please review.

File: app/message-pin/client/pinMessage.js [hunk 1]
```diff
@@ -1,42 +1,50 @@
...
+import { TAPi18n } from 'meteor/tap:i18n';
 
 Meteor.methods({
 	pinMessage(message) {
 		if (!Meteor.userId()) {
+			toastr.error(TAPi18n.__('error-pinning-message'));
 			return false;
-		}
-		if (!settings.get('Message_AllowPinning')) {
+		} else if (!settings.get('Message_AllowPinning')) {
```
Comment (ggazzo) [inline app/message-pin/client/pinMessage.js]: we dont need an `else` statement here, as we have a `return` inside each if

Comment (ashwaniYDV) [reply app/message-pin/client/pinMessage.js]: @ggazzo I had done this PR in gsoc19 time and thus lost my old repository. I had made a new PR this year for the same issue. Sorry for the inconvenience.

#New PR #16056

**Comment by ashwaniYDV** [comment]
@ggazzo Would you please review this PR #16056.

**Pull request closed without merging**